.github/workflows/markdown.links.config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Can you add the `$schema` corresponding to this config?

Reply:
Unfortunately [markdown-link-check](https://github.com/tcort/markdown-link-check) does not provide a `$schema`